cms/models/pagemodel.py

```diff
@@ -52,7 +53,7 @@ class TreeNode(MP_Node):
 
     class Meta:
         app_label = 'cms'
-        ordering = ('path',)
+        ordering = ['path']
```

Comment:
I might be overlooking something, but can't `TreeNode` be removed. I'm not sure what a custom app would do with it. @marksweb What do you think? Remove or keep for compatibility?